lib/publiccloud/instance.pm

```diff
@@ -85,6 +91,37 @@ sub run_ssh_command {
     }
 }
 
+=head2 retry_ssh_command
+
+    ssh_script_retry(command[, retry => 3][, delay => 10][, timeout => 90][, ssh_opts =>'..'][, username => 'XXX'][, no_quote => 0]);
+
+Run a C<command> via ssh in the given PC instance until it succeeds or
+the given number of retries is exhausted and an exception is thrown.
+Timeout can be set by C<timeout> or 90 sec by default.
+By default, the command is passed in single quotes to SSH.
+To avoid quoting use C<<no_quote=>1>>.
+With C<<ssh_opts=>'...'>> you can overwrite all default ops which are in
+C<<$instance->ssh_opts>>.
+Use argument C<username> to specify a different username then
+C<<$instance->username()>>.
+=cut
+sub retry_ssh_command {
+    my ($self, $cmd, %args) = @_;
+    $cmd = $args{cmd} if (!defined $cmd && $args{cmd});
+    $args{rc_only} = 1;
+    $args{cmd_timeout} //= 90;    # Timeout before we cancel the command
+    $args{timeout}     //= 95;    # Hard timeout before we consider something going horribly wrong
+    my $tries = $args{retry} // 3;
+    my $delay = $args{delay} // 10;
```

Comment:
nit thing...
```suggestion
    my $tries = delete $args{retry} // 3;
    my $delay = delete $args{delay} // 10;
```
maybe this, as you don't wanna pass it to `run_ssh_command()` otherwise there is no need for `$delay` variable.